source/includes/api/_metadata.md

```diff
@@ -1,188 +1,436 @@
+                    "learn_more_link": "http://data.worldbank.org/indicator/SH.H2O.SAFE.ZS",
+                    "source_organization_link": "null",
+                    "source_organization": "United Nations World Health Organization (UN WHO)/World Health Organization and United Nations Children's Fund Joint Monitoring Programme for Water Supply and Sanitation (WHO UNICEF JMP WSS)",
+                    "function": "This data set displays the percentage of urban population with access to an improved drinking water source in a dwelling or located within a convenient distance from the user's dwelling.",
+                    "functions": "This data set displays the percentage of urban population with access to an improved drinking water source in a dwelling or located within a convenient distance from the user's dwelling.",
+                    "name": "Urban Population with Access to an Improved Water Source",
+                    "technical_title": "Percentage of Urban Population with Access to an Improved Water Source (WHO/UNICEF)"
+                },
+                "columns": {
+                    "year": {
+                        "description": "Measured year",
+                        "alias": "Year"
+                    },
+                    "value": {
+                        "alias": "Value"
+                    },
+                    "country": {
+                        "alias": "Country Name"
+                    },
+                    "iso": {
+                        "alias": "ISO-3 Country Code"
+                    }
+                },
+                "createdAt": "2017-07-07T12:49:48.721Z",
+                "updatedAt": "2017-08-28T14:30:17.701Z",
+                "status": "published"
+            }
+        }
+    ]
+}
+```
 
-Any of these conditions are not met, the API will generate a 403 `Forbidden` error message.
+## Updating a metadata object
 
-When updating a metadata, you must always specify both `language` and `application`, those will be used to determine which metadata entity will be updated.
+As for creation, the endpoints for updating metadata objects follow the `dataset/<dataset_id>`, `layer/<layer_id>`, and `widget/<widget_id>` style. You need to provide your authorization token (`<auth_token>`) in the header, and PATCH a JSON object that must contain valid values for the fields `application` and `language`. If the object is correctly updated a `200` response is returned, as well as the updated metadata object. If the user is not authorized to update metadata objects a `403` error is returned; see the required [credentials](#overview-of-available-endpoints). If the request is invalid a `400` error is returned; potentially with a more detailed error message.
```

Comment:
Same comments here as in the "Creating a metadata object" section about auth and errors.